[BUG] aenum based unit tests seem broken

### Describe the bug

When following the typesafeapi document, I got to the step where one needs to run tests on the infrastructure.  And it appears the aenum package has failing unit tests. 

### Expected Behavior

I expected the infrastructure to build and run tests with a success code.

### Current Behavior

 Here is the errors I get:

`============================================================ short test summary info ============================================================
FAILED cdk.out/asset.da8c7c102fcd20cecb3593d5a2b396a3ed3e536a31ccf35c312daaea6fd21e81/aenum/test_v3.py::TestEnumV3::test_class_nested_enum_and_pickle_protocol_four - NameError: name 'test_pickle_dump_load' is not defined
FAILED cdk.out/asset.da8c7c102fcd20cecb3593d5a2b396a3ed3e536a31ccf35c312daaea6fd21e81/aenum/test_v3.py::TestEnumV3::test_pickle_enum_function_with_qualname - NameError: name 'test_pickle_dump_load' is not defined
FAILED cdk.out/asset.da8c7c102fcd20cecb3593d5a2b396a3ed3e536a31ccf35c312daaea6fd21e81/aenum/test_v3.py::TestEnumV3::test_subclasses_with_getnewargs_ex - NameError: name 'test_pickle_dump_load' is not defined
FAILED cdk.out/asset.da8c7c102fcd20cecb3593d5a2b396a3ed3e536a31ccf35c312daaea6fd21e81/aenum/test_v3.py::TestStackoverflowAnswersV3::test_arduino_headers - NameError: name 'tempdir' is not defined
FAILED cdk.out/asset.da8c7c102fcd20cecb3593d5a2b396a3ed3e536a31ccf35c312daaea6fd21e81/aenum/test_v3.py::TestExtendEnumV3::test_extend_enum_shadow_property_stdlib - AttributeError: <enum 'Color'> has no attribute 'value'. Did you mean: 'blue'?
`

### Reproduction Steps

I followed the type_safe_api guidelines, and after adding the 'SayGoodbye' api call I ran `pdk build` to get the above error.

### Possible Solution

_No response_

### Additional Information/Context

_No response_

### PDK version used

0.22.41

### What languages are you seeing this issue on?

Python

### Environment details (OS name and version, etc.)

macos, ventura 13.6

--- a/packages/infrastructure/src/projects/python/infrastructure-py-project.ts
+++ b/packages/infrastructure/src/projects/python/infrastructure-py-project.ts
@@ -118,12 +118,16 @@ export class InfrastructurePyProject extends AwsCdkPythonApp {
         ),
     };
 
+    const tstDir = "tests";
+
     options.sample !== false &&
       this.emitSampleFiles(srcDir, [this.moduleName], mustacheConfig);
     options.sample !== false &&
-      this.emitSampleFiles(testDir, ["tests"], mustacheConfig);
+      this.emitSampleFiles(testDir, [tstDir], mustacheConfig);
 
-    this.testTask.reset("poetry run pytest ${CI:-'--snapshot-update'}");
+    this.testTask.reset(
+      `poetry run pytest ${tstDir}/ \${CI:-'--snapshot-update'}`
+    );
   }
 
   private emitSampleFiles(

--- a/packages/type-safe-api/src/project/codegen/handlers/generated-python-handlers-project.ts
+++ b/packages/type-safe-api/src/project/codegen/handlers/generated-python-handlers-project.ts
@@ -63,7 +63,9 @@ export class GeneratedPythonHandlersProject extends PythonProject {
       // Pytest fails with exit code 5 when there are no tests.
       // We want to allow users to delete all their tests, or to upgrade an existing project without breaking their build
       // See: https://github.com/pytest-dev/pytest/issues/2393
-      this.testTask.reset("pytest || ([ $? = 5 ] && exit 0 || exit $?)");
+      this.testTask.reset(
+        `pytest ${this.tstDir}/ || ([ $? = 5 ] && exit 0 || exit $?)`
+      );
     }
 
     [

